src/app_model/expressions/_expressions.py

```diff
@@ -119,6 +95,29 @@ def safe_eval(expr: Union[str, bool, Expr], context: Optional[Mapping] = None) -
 class Expr(ast.AST, Generic[T]):
     """Base Expression class providing dunder and convenience methods.
 
+    This is a subclass of `ast.AST` that provides rich dunder methods that
+    facilitate joining and comparing typed expressions.  It only implements a
+    subset of ast Expressions (for safety of evaluation), but provides more
+    than `ast.literal_eval`.
+
+    Expressions that are supported:
+    - Names: 'myvar' (these must be evaluated along with some context)
+    - Constants: '1'
+    - Comparisons: 'myvar > 1' (includes `in`, `not in`)
```

Comment:
The include isn't actually correct... we _dont_ support the `in` operator (that would be a `__contains__` method).  So someone would specifically need to use the special `in_` and `not_in` methods on these expression objects (which is a funnier syntax).  If that doesn't make sense, probably best just to remove the parenthetical addition here

Reply:
My bad, did not read properly. I will remove it.

Just for info, I found that `in` did seem to work:
```python
from app_model.expressions import parse_expression

a = parse_expression('1 in myvar')
a.eval({'myvar': (1,2)})
```
gives `True`. When you say we don't support it, do you mean it will 'work' but shouldn't be used as it behaves weird?